.github/workflows/build.yaml

```diff
@@ -0,0 +1,47 @@
+name: Build baselibs
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  build_baselibs:
+    runs-on: ubuntu-latest
```

Comment:
One more question, @pchakraborty, as I go through my PRs. Why does this run on ubuntu but use EPEL?